docs/release_notes/feb2023.md

```diff
@@ -8,6 +8,69 @@ tags:
   - changelog
 ---
 
+## 2.8.0.0 (February 28, 2023)
+
+- Prophecy Python libs version: 1.4.2
+- Prophecy Scala libs version: 7.0.7
+
+### Updates {#UpdatesRelease280}
+
+#### No team required on sign up
+
+Previously, A user was asked to enter a unique team when signing up for Prophecy. To improve the sign-up experience, this requirement has been removed
+Now the user can just sign up by giving their Name, Email and Company name. They can be added to a team by Admin after signing up.
+
+![New_Sign_up](img/new_sign_up.png)
+
+#### Sorting on tables
+
+All metadata tables will now have options to sort on the columns. By default, the entities in table will be sorted on `Modified At` or `Created At` time in decreasing order.
+
+![Sorting](img/Sorting.gif)
+
+#### Use a Cluster in Current tab
+
+A User editing Prophecy Pipelines in multiple tabs can now detach the cluster Attached to a Pipeline in another tab and attach the current Pipeline by clicking the same cluster.
+They will see this warning before the cluster is detached in previous tab.
+![Attach-cluster](img/Attach-cluster1.png)
+
+If there were running Jobs in another tab, they will be prompted to cancel the existing Jobs before switching.
+![Attach-cluster-warning](img/Attach_cluster_job_warning.png)
+
+#### Link to Databricks Workflow in Released Prophecy Jobs
+
+In Job editor view we've added the link to the corresponding Databricks Workflow.
+
+Please note: this will only appear if the Job is enabled and released.
+
+![Databricks-link](img/Databricks_link.png)
+
+#### Create Pipelines and Jobs directly from Project Browser
+
+Previously, a user had to navigate out of Pipeline editor to add a new Job or Pipeline. Now they can do it via the same screen and shown below. Please note, the Pipelines and Jobs can only be created in same Project and Git branch using this option.
+
+![Create-pipeline](img/create_pipeline.gif)
+
+#### Alert user on Databricks Token Expiry
+
+If their Databricks token is expired or is invalid, the user will now see a prompt as below to update their Databricks token. They will be prompted on the Jobs editor or Pipeline editor when attaching to the Fabric.
+
+![token-expiry](img/Databricks-token-expiry.png)
+
+### Improvements {#ImprovementsRelease280}
+
+#### Allow Run on main branch
+
+Editing your Pipeline on Main branch is not allowed. Interactive run and Partial run on Gems is allowed when viewing a Pipeline on Main branch. Interims will also work as expected.
+
+![Run_main_branch](img/Run_Main_branch.png)
+
+#### Cluster restart after setting dependencies
+
+Improved the cluster restart behaviour when adding a new Pipeline dependency. When a new Pipeline dependency is added, cluster will be detached and prompt for a restart as shown below. A small subset of users were experiencing cluster restart loops when using certain dependencies and this has been addressed
```

Comment:
I suggest to remove this text: 
 "A small subset of users were experiencing cluster restart loops when using certain dependencies and this has been addressed"

Reply:
I suggested adding that, @atbida. What's your thoughts on the matter?